### ISSUE TITLE
Provide a script to get OCR result for RVL-CDIP in layoutLM

### ISSUE THREAD

**Issue by yaoliUoA** [microsoft/unilm#216]
**Describe**
I am using LayoutLM, would you please provide the script to prepare the OCR output (html format) for the RVL-CDIP? The readme mentions about Tesseract, however it will be much conventient if the script can be provided.

**Comment by Lambert-Shirzad** [comment]
Yes, the trained model is not that useful and experiments reported in the paper are not that repeatable if the pre-processing step for an input image needs to be reverse engineered.

**Comment by hazoth** [comment]
need ocr script too. especially for the parameters to run tesseract. if possible, the processed hocr files would be better please. i can not repeat the same result in the paper now.

**Comment by siatwangmin** [comment]
Yes, it is hard to repeat the result if we can't get the processed data, can you release the data?

**Comment by greeneggsandyaml** [comment]
Yes, this would be very helpful for our research!

**Comment by Yazooliu** [comment]
Yes，also need the  OCR output (html format)  or scipit to understand the inputs(such as layout info, bbox position info, etc) thanks
